packages/smooth_app/lib/data_models/user_management_provider.dart

```diff
@@ -39,8 +40,14 @@ class UserManagementProvider with ChangeNotifier {
 
   /// Mounts already stored credentials, called at app startup
   Future<void> mountCredentials() async {
-    final String? userId = await DaoSecuredString.get(_USER_ID);
-    final String? password = await DaoSecuredString.get(_PASSWORD);
+    final String? userId;
+    final String? password;
+    try {
+      userId = await DaoSecuredString.get(_USER_ID);
+      password = await DaoSecuredString.get(_PASSWORD);
+    } on PlatformException {
```

Comment:
Could you add a comment here on the kind of exception it was (SSL related, possibly on old devices ?)

Reply:
Added a comment